OpenRA.Mods.Cnc/Traits/Mine.cs

```diff
@@ -59,11 +59,15 @@ bool ICrushable.CrushableBy(Actor self, Actor crusher, BitSet<CrushClass> crushC
 			return info.CrushClasses.Overlaps(crushClasses);
 		}
 
-		bool ICrushable.TryCalculatePlayerBlocking(Actor self, BitSet<CrushClass> crushClasses, out LongBitSet<PlayerBitMask> blocking)
+		LongBitSet<PlayerBitMask> ICrushable.CrushableBy(Actor self, BitSet<CrushClass> crushClasses)
 		{
-			// Fall back to the slow path
-			blocking = default(LongBitSet<PlayerBitMask>);
-			return false;
+			var blocking = default(LongBitSet<PlayerBitMask>);
+			if (info.BlockFriendly)
+				blocking = self.Owner.AllyMask;
```

Comment:
Imho `var blocking = info.BlockingFriendly ? self.Owner.AllyMask : default(LongBitSet<PlayerBitMask>);` is a bit better.

Reply:
Done!